src/scripts/main.js

```diff
@@ -1,5 +1,50 @@
 'use strict';
 
-document.addEventListener('click', e => {
-  // write code here
+const spider = document.querySelector('.spider');
+const spiderParent = spider.parentElement;
+
+spiderParent.addEventListener('click', (e) => {
+  const parentRect = spiderParent.getBoundingClientRect();
+
+  const style = getComputedStyle(spiderParent);
+  const borderLeftWidth = parseFloat(style.borderLeftWidth);
+  const borderTopWidth = parseFloat(style.borderTopWidth);
+  const borderRightWidth = parseFloat(style.borderRightWidth);
+  const borderBottomWidth = parseFloat(style.borderBottomWidth);
+
+  const maxLeft
+    = parentRect.width
+      - spider.offsetWidth
```

Comment:
```suggestion
  const maxLeft = parentRect.width
    - spider.offsetWidth
```
![image](https://github.com/mate-academy/js_task-move-spider-DOM/assets/96409563/f0eb87bf-b93d-4b54-8751-340317c3d430)